src/web/modules/gateway_accounts/gateway_accounts.http.ts

```diff
@@ -152,17 +152,23 @@ const writeAccount = async function writeAccount(req: Request, res: Response): P
 
   logger.info(`Created new Gateway Account ${gatewayAccountIdDerived}`)
 
-  // connect system linked services to the created account
+  // connect system linked services to the created account  
   if (linkedService) {
     await AdminUsers.updateServiceGatewayAccount(
       linkedService,
       gatewayAccountIdDerived
     )
     logger.info(`Service ${linkedService} linked to new Gateway Account ${gatewayAccountIdDerived}`)
+    
+    const serviceDetails = await AdminUsers.service(linkedService)
+    const isServiceAlreadyLive = serviceDetails.current_go_live_stage === 'LIVE'
```

Comment:
would it be clearer to have the following

```
const updateServiceToLive = account.isLive() && serviceDetails.current_go_live_stage !== 'LIVE'
await AdminUsers.updateServiceDetails(linkedService, updateServiceToLive, account.sector, account.internalFlag)
```

Reply:
Good idea - will do that

Reply:
Done

Reply:
I think the `isServiceAlreadyLive` const is now unused

Reply:
Thanks - good spot

Reply:
Done